common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -0,0 +1,228 @@
+"""
+Acceptance tests for the Import and Export pages
+"""
+from abc import abstractmethod
+from bok_choy.promise import EmptyPromise
+from .base_studio_test import StudioLibraryTest, StudioCourseTest
+from ...fixtures.course import XBlockFixtureDesc
+from ...pages.studio.import_export import ExportLibraryPage, ExportCoursePage, ImportLibraryPage, ImportCoursePage
+from ...pages.studio.library import LibraryEditPage
+from ...pages.studio.container import ContainerPage
+from ...pages.studio.overview import CourseOutlinePage
+
+
+# pylint: disable=no-member
+class ExportTestMixin(object):
+    """
+    Tests to run both for course and library export pages.
+    """
+    def test_export(self):
+        good_status, is_tarball_mimetype = self.export_page.download_tarball()
+        self.assertTrue(good_status)
+        self.assertTrue(is_tarball_mimetype)
+
+
+class TestCourseExport(ExportTestMixin, StudioCourseTest):
+    """
+    Export tests for courses.
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(TestCourseExport, self).setUp()
+        self.export_page = ExportCoursePage(
+            self.browser,
+            self.course_info['org'], self.course_info['number'], self.course_info['run'],
+        )
+        self.export_page.visit()
+
+
+class TestLibraryExport(ExportTestMixin, StudioLibraryTest):
+    """
+    Export tests for libraries.
```

Comment:
Same note as above, and make it generic since it works for both libraries and courses.